gammapy/spectrum/energy.py

```diff
@@ -0,0 +1,293 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function, division
+from astropy.units import Quantity
+from astropy.io import fits
+from ..utils.array import array_stats_str
+
+import astropy.units as u
+import numpy as np
+
+__all__ = ['Energy',
+           'EnergyBounds']
+
+class Energy(u.Quantity):
+
+    """Energy bin centers.
+
+    Stored as "ENERGIES" FITS table extensions.
+
+    Parameters
+    ----------
+    energy : `~numpy.array`, scalar, `~astropy.units.Quantity`
+        Energy
```

Comment:
Remove these empty lines between parameters to save some screen space and be consistent throughout Gammapy.